site/content/docs/5.3/components/title-bars.md

```diff
@@ -19,31 +19,31 @@ See them in action in our [examples page]({{<docsref "/examples/title-bars">}}).
 
 ## Background color
 
-To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-dark`, `.bg-white` and `.bg-supporting-*`.'
+To change the appearance of a title bar, use our [background utilities]({{<docsref "/utilities/background">}}). Orange branded background utilities are `.bg-dark`, `.bg-white` and `.bg-supporting-*`.
 
-Title bars can be displayed with black or white backgrounds. When using black or white backgrounds, illustrations can be included as an option, but it's not necessary.
+Title bars can be displayed with `.bg-black`, `.bg-dark`, `.bg-white` or no background. When using `.bg-black`, `.bg-dark`, `.bg-white` or no background, illustrations can be included as an option, but it's not necessary.
 Supporting colors can also be used as the background color for title bars, but not the supporting light and dark tint colors. When using one of the supporting colors, illustrations must be used.
 
-The image should not overlap on title and the title shouldn't wrap. If this happens, please **transform your title bar** into a white/black one without any image.
+The image should not overlap on title and the title shouldn't wrap. If this happens, please **transform your title bar** into a white/dark one without any image.
```

Comment:
This sentence is also to be refined since they are no more "white/black" nor "white/dark" title bars.

Reply:
Tackled in https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/2350/commits/c176667a9debd7ca383d06b95c133da75ba18e9c